food-truck-frontend/src/pages/search/truck.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import {Button, Divider, FormGroup, Grid, List, ListItem, TextField} from "@material-ui/core";
+import api from "../../util/api";
+import SearchTruckRow from "../../components/search/SearchRow";
+import jssPluginPropsSort from "jss-plugin-props-sort";
+
+const truck = [{
+    id: -1,
+    name: "",
+    description: ""
+}]
```

Comment:
What is this for? If it's just to initialize this.state.trucks, why not initialize it to []?